dask/bytes/tests/test_hdfs.py

```diff
@@ -297,8 +305,8 @@ def test_glob(hdfs):
             {basedir + p for p in ['/a', '/a1', '/a2', '/a3', '/b1', '/c', '/c2']})
 
 
-def test_distributed(hdfs):
-    dist = pytest.importorskip('distributed')
+@pytest.mark.skipif(not distributed)
```

Comment:
Seems the issue is `flake8` wants to complain about this line. Though this falls out of the function having the `loop` variable and a decorator.